.travis.yml

```diff
@@ -10,6 +10,8 @@ before_install:
   - curl -L https://github.com/docker/compose/releases/download/1.22.0/docker-compose-`uname -s`-`uname -m` > docker-compose
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
+  # fetch full history for correct current and previous version detection
+  - git fetch --unshallow
```

Comment:
is this something we need to change also in other projects?

Reply:
Yes, for the projects that use `previousStableVersion` to derive `mimaPreviousArtifacts`.

Reply:
According to the GitHub code search the only other project that used `previousStableVersion` is Alpakka. The fix is here: https://github.com/akka/alpakka/pull/1305

https://github.com/search?q=org%3Aakka+previousStableVersion&type=Code
https://github.com/search?q=org%3Alightbend+previousStableVersion&type=Code